regulations/static/regulations/css/less/module/comment.less

```diff
@@ -4,6 +4,40 @@ comment.less
 comment.less contains styles related to commenting on sections
 */
 
+#preamble-read {
+  margin: 0;
+  padding: 0;
+}
+
+.preamble-header {
+  position: absolute;
+  right: 0;
+  z-index: 2000;
```

Comment:
Just curious--do define z-index variables anywhere? Setting z-indices across the app can get hard to manage.